src/main.rs

```diff
@@ -296,9 +296,14 @@ fn get_metadata(messages: &[Message]) -> CTRConfig {
         _ => artifact.target.name,
     };
 
+    let author = match std::env::var("CARGO_PKG_AUTHORS") {
```

Comment:
Is there any difference between this and just a `package.authors.is_empty()` or something? It seems like [from the docs](https://doc.rust-lang.org/cargo/commands/cargo-metadata.html#output-format) that there would still be an empty list if the field wasn't specified? 

I think the env var seems fine as well if it works, but I was under the impression that cargo [doesn't set](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-3rd-party-subcommands) these kind of vars for subcommands.

Reply:
Well it does work, and as I said it is even suggested in the RFC.

The vector is still valid when length is zero, but this approach simplifies the parsing (as we would either have to choose which index to use or concatenate the strings) and the future stability, as it is implied `cargo-metadata` may have breaking changes in the future, as well as many other packages.

Reply:
Ok, sounds fine then. I couldn't find any `cargo-metadata` issues about the author field but it does seem like it might be deprecated, so the env var seems like a good option if it works.